docs/src/autodiff/operator_overloading.md

```diff
@@ -0,0 +1,30 @@
+# Operator Overloading
+
+## Examples
+
+### ForwardDiffZero
+
+````@eval
+using Markdown
+code = read(joinpath(@__DIR__,"../../../test/demos/forwarddiffzero.jl"), String)
+code = replace(code, raw"$" => raw"\$")
+Markdown.parse("""
+```julia
+$(code)
+```
+""")
+````
+
+### ReverseDiffZero
+
+````@eval
+using Markdown
+code = read(joinpath(@__DIR__,"../../../test/demos/reversediffzero.jl"), String)
+code = replace(code, raw"$" => raw"\$")
+Markdown.parse("""
+```julia
+$(code)
+```
+""")
+````
+
```

Comment:
the code in this section is gross but its the current best way to just insert code into the source.
see https://github.com/JuliaDocs/Documenter.jl/issues/499

Note that messing with the indentation to anything other than this completely breaks it